benefits/core/views.py

```diff
@@ -20,11 +20,6 @@ def _index_content_title():
     return _("core.pages.index.content_title")
 
 
-def _index_image():
-    """Helper returns a viewmodels.Image for the common index page."""
-    return viewmodels.Image("ridertappingbankcard.png", pgettext("image alt text", "core.pages.index.image"))
```

Comment:
This made me realize that we weren't and aren't using the `Image.alt` value in `page.html`. 🤔 Unrelated to this ticket though.

Reply:
It's true - this is [implemented as a CSS `background` property](https://github.com/cal-itp/benefits/blob/dev/benefits/core/templates/core/page.html#L16) rather than an `<img>` tag, which is why there is no alt attribute (and how the Accessibility checker passes).

If you can find anything related to using images in CSS background and alt attributes, or maybe there's a special `aria-` attribute we can use instead? Go ahead and file an issue since yeah, we weren't using it before either, and shouldn't prevent this PR from being merged.